releases/vexflow-debug.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
Yes, I also think so that it's right way that you only do bump and publish releases, but this time was special.
For making travis-ci work, I had to update releases because current version uses the latest release as a generating blessed images script.
Pros for this commits are that contributors don't need to handle conflict of managing blessed svg images or cache.

This time, I ran all the `grunt publish` tasks except `grunt release`.